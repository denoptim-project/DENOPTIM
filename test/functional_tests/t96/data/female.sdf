> <GraphENC>
0 0_0_0_-1,1_0_1_1,
0_0_1_1_c:0_c:0,
