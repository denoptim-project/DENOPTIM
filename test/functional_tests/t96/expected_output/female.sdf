
  CDK     02232113173D

  6  5  0  0  0  0  0  0  0  0999 V2000
    0.0000    0.0000    0.0000 C   0  0  0  0  0  5  0  0  0  0  0  0
    0.0000    0.0000    0.0000 C   0  0  0  0  0  6  0  0  0  0  0  0
    1.0000    0.0000    0.0000 H   0  0  0  0  0  0  0  0  0  0  0  0
    0.0000    0.0000    0.0000 C   0  0  0  0  0  5  0  0  0  0  0  0
    0.0000    0.0000    0.0000 C   0  0  0  0  0  6  0  0  0  0  0  0
    1.0000    0.0000    0.0000 H   0  0  0  0  0  0  0  0  0  0  0  0
  2  3  1  0  0  0  0
  1  2  1  0  0  0  0
  5  6  1  0  0  0  0
  4  5  1  0  0  0  0
  2  5  1  0  0  0  0
M  END
> <GCODE>
1

> <GraphENC>
1 13_ABS_VERTEX,18_ABS_VERTEX, 13_0_18_0_1_c:0_c:0,

> <GraphMsg>


$$$$
