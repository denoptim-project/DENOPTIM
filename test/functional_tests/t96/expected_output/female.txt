 *6 +5.*\n((.*C.*\n){2}.*H.*\n){2}.*2.*3.*1.*\n.*1.*2.*1.*\n.*5.*6.*1.*\n.*4.*5.*1.*\n.*2.*5.*1.*\n
